v_4_8_0/files/k8s-resource/calico-all.yaml

```diff
@@ -462,19 +660,19 @@ spec:
               cpu: 250m
               memory: 150Mi
           livenessProbe:
-            httpGet:
-              path: /liveness
-              port: 9099
-              host: localhost
+            exec:
+              command:
+              - /bin/calico-node
+              - -felix-live
```

Comment:
Not sure about this one.

Reply:
I'm not either. Shows up in [v3.9](https://docs.projectcalico.org/v3.9/manifests/calico.yaml). [v3.8](https://docs.projectcalico.org/v3.8/manifests/calico.yaml) uses the old `httpGet` way. I can do some research on pros and cons. Should this also be discussed with SIG-Infra?